float32[] positions
float32[] joints

bool motor_enabled
bool arm_normal
bool arm_busy
string error

uint8 speed

uint8 tank_psi
